keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -0,0 +1,428 @@
+---
+title: Object Bucket Provisioning
+authors:
+  - "@jeffvance"
+  - "@copejon"
+owning-sig: "sig-storage"
+reviewers:
+  - "@saad-ali"
+  - "@alarge"
+  - "@erinboyd"
+  - "@guymguym"
+  - "@travisn"
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2019-11-25
+last-updated: 2020-07-08
+status: provisional
+---
+
+# Object Bucket Provisioning
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+  - [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+  - [Vocabulary](#vocabulary)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+      - [Admin](#admin)
+      - [User](#user)
+  - [APIs](#apis)
+    - [Storage APIs](#storage-apis)
+      - [BucketRequest](#bucketrequest)
+      - [Bucket](#bucket)
+      - [BucketClass](#bucketclass)
+    - [Access APIs](#access-apis)
+      - [BucketAccessRequest](#bucketaccessrequest)
+      - [BucketAccess](#bucketaccess)
+      - [BucketAccessClass](#bucketaccessclass)
+  - [Provisioner Secrets](#provisioner-secrets)
```

Comment:
After API added section on typical create and delete workflows for green- and brownfield.

Reply:
Add TOC for gRPC spec